src/renderer/components/panel.ts

```diff
@@ -238,8 +253,17 @@ export default class ComponentPanel {
     }
   }
 
-  land_enter() {
-    this.counter = FRAME_LAND.length
+  land_enter()   { this.counter = FRAME_LAND.length }
+  land_execute() { 
+    if (this.counter <= 0) { 
+      if (this.under === blank ? false : this.under.state === HANG) {
```

Comment:
We might want this is its on function so we can document it but its easily readable I dont think its a big deal.